### DISCUSSION TITLE
Not using GOOGLE_REVERSE_PROXY config from .env

### DISCUSSION THREAD

**Discussion by AdiRishi** [danny-avila/LibreChat#3951] [Troubleshooting] [answered]
### What happened?

Using `LibreChat v0.7.5-rc1`

I've been setting up Cloudflare AI Gateway for all the various API endpoints.
Both `OPENAI_REVERSE_PROXY` and `ANTHROPIC_REVERSE_PROXY` worked perfectly.

But it looks like `GOOGLE_REVERSE_PROXY` is being totally ignored.

### Steps to Reproduce

1. Set `GOOGLE_REVERSE_PROXY=this-is-a-bad-url` in `.env`
2. Re run `docker compose up -d`

You will notice that Google still works perfectly (using gemini-pro model), meaning that this config option was totally ignored.
I confirmed that the env file was copied correctly as when I run `docker compose exec api sh` and check the `.env` file in there it has the correct value.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
This is not implemented for Google natively

**Reply by AdiRishi** [reply]
Holy wow that was a quick response 🤯 

So just to confirm, you're saying it is currently not possible to use reverse proxy endpoints for google at the moment? Curious if this is a planned feature or not.

**Reply by danny-avila** [reply]
I will add it, it was initially not possible with the Google SDK when first implemented, but looks like it is now: https://github.com/google-gemini/generative-ai-js/issues/25

**Reply by AdiRishi** [reply]
Awesome. No hurry, I can't even image how busy you must be maintaining this project.
Just waned to say, it's so incredible what you've made here!

**Reply by KiGamji** [reply]
@danny-avila could you add it to your milestone please? that would be very great. Thanks in advance. Also, please consider adding `GOOGLE_SAFETY_CIVIL_INTEGRITY`
